.github/workflows/build-and-publish.yml

```diff
@@ -28,6 +28,10 @@ jobs:
           latest: 1   # this should only be set on the current minor branch
           github_actor: ${{ secrets.LNB_ACTOR }}
           github_token: ${{ secrets.LNB_TOKEN }}
+          sign_modules: 1
```

Comment:
trivial: The README.md (in this PR) says:
```
The following values should be added to github action workflows that wish to
utilize the signing behavior.

> sign_modules: ON
```

this should probably match and use "ON" rather than 1.